bindinfo/handle.go

```diff
@@ -380,7 +377,10 @@ type tmpBindRecordMap struct {
 	flushFunc func(record *BindRecord) error
 }
 
+// flushToStore calls flushFunc for items in tmpBindRecordMap and removes them with a delay.
 func (tmpMap *tmpBindRecordMap) flushToStore() {
+	tmpMap.Lock()
```

Comment:
is this a bugfix?

Reply:
No, currently there is only one goroutine that would write this map, except for the `admin flush bindings` which is only used in test. This PR add lock to prevent potential bugs in future.